uint64 serial # serial id of reference poses

float64 sim_t # Time interval of reference poses

float64 tgt_vel # target speed
float64 tgt_acc # target acceleration
float64 effort # effort mode